Library/UITraitCollection.swift

```diff
@@ -0,0 +1,18 @@
+import UIKit
+
+public extension UITraitCollection {
+  public func ksr_isAccessibilityCategory() -> Bool {
```

Comment:
How about a `switch` and early `return`?

```swift
  public func ksr_isAccessibilityCategory() -> Bool {
    let sizeCategory = self.preferredContentSizeCategory

    if #available(iOS 11, *) {
      return sizeCategory.isAccessibilityCategory
    }

    switch sizeCategory {
      case .accessibilityMedium,
           .accessibilityLarge,
           .accessibilityExtraLarge,
           .accessibilityExtraExtraLarge,
           .accessibilityExtraExtraExtraLarge:
      return true
    default:
      return false
    }
  }
```

Reply:
My thinking here was simple `2 returns` vs `3 returns` 😄 Happy to change, just let me know

Reply:
I guess I just don't like all the `||`'s 😄 

You know my favourite would be:

```swift
  public func ksr_isAccessibilityCategory() -> Bool {
    let sizeCategory = self.preferredContentSizeCategory

    if #available(iOS 11, *) {
      return sizeCategory.isAccessibilityCategory
    } else {
      return [
        .accessibilityMedium,
        .accessibilityLarge,
        .accessibilityExtraLarge,
        .accessibilityExtraExtraLarge,
        .accessibilityExtraExtraExtraLarge
      ]
      .contains(sizeCategory)
    }
  }
```

Do it for fun :trollface:

Reply:
I can do that. Just wondering if the contains wouldn’t be more expensive that OR? 🤔

Reply:
Doubt it would be noticeable, even if it was, it's only iOS 10 users 😄 

By the way we can also lose the `else` and just return 👍